webapp/i18n/es.json

```diff
@@ -4,6 +4,7 @@
   "plugin.upload": "Subir usando el plugin demo",
   "root.clicktoclose": "Haga clic en cualquier lugar para cerrar.",
   "root.triggered": "Has activado el componente raíz del complemento de demostración.",
+  "rhs.triggered": "Has activado el componente del panel derecho del complemento de demostración",
```

Comment:
We might not want this file changed in this PR. Generally, PRs only change `en.json` and the other files are updated using the translation server. That may work differently for plugins though, so @crspeller would know better about it

Reply:
It's fine here. There isn't a translation process for plugins yet.